ConsoleApp/Program.cs

```diff
@@ -0,0 +1,184 @@
+﻿// 04 - Methods
+// HOMEWORK
+// Not your boring middle school homework — these tasks will help you in real life! Have fun with your assignment for this topic:
+//
+// Define and call with different parameters next methods:
+//
+// 1. Method that will return max value among all the parameters
+// 2. ....................... min value ........................
+// 3. Method TrySumIfOdd that accepts 2 parameters and returns true if sum of numbers between inputs is odd, otherwise false, sum return as out parameter
+// 4. Overload first two methods with 3 and 4 parameters
+// 
+// Extra:
+// Define and call with different parameters next methods:
+//
+// 1. Method Repeat that will accept string X and number N and return X repeated N times (e.g. Repeat(‘str’, 3) returns ‘strstrstr’ = ‘str’ three times)
+
+internal class Program
+{
+    static double Define_Max(int num, long key)
```

Comment:
strange naming for variables, they all mean the same, but all names are different. I suggest better `value1`, `value2`, ...